src/dsync/dsync.c

```diff
@@ -3047,7 +3049,7 @@ int main(int argc, char **argv)
 
     while (1) {
         int c = getopt_long(
-            argc, argv, "b:cDso:LPSvqh",
+            argc, argv, "b:cDe:so:LPSvqh",
```

Comment:
Do you mean to have ``X:`` in here instead of ``e:``?

Reply:
Yes